met/src/tools/core/stat_analysis/aggr_stat_line.cc

```diff
@@ -2627,17 +2619,20 @@ void aggr_ecnt_lines(LineDataFile &f, STATAnalysisJob &job,
       v                           = it->second.mse_oerr_na.wmean(it->second.ens_pd.wgt_na);
       it->second.ens_pd.rmse_oerr = (is_bad_data(v) ? bad_data_double : sqrt(v));
 
-      crps_fcst  = it->second.ens_pd.crps_na.wmean(it->second.ens_pd.wgt_na);
-      crps_climo = it->second.crps_climo_na.wmean(it->second.ens_pd.wgt_na);
+      crps_emp    = it->second.ens_pd.crps_emp_na.wmean(it->second.ens_pd.wgt_na);
+      crpscl_emp  = it->second.crpscl_emp_na.wmean(it->second.ens_pd.wgt_na);
+      crps_gaus   = it->second.ens_pd.crps_gaus_na.wmean(it->second.ens_pd.wgt_na);
+      crpscl_gaus = it->second.crpscl_gaus_na.wmean(it->second.ens_pd.wgt_na);
 
-      // Compute aggregated CRPSS
-      if(!is_bad_data(crps_fcst) && !is_bad_data(crps_climo) &&
-         !is_eq(crps_climo, 0.0)) {
-         it->second.ens_pd.crpss = (crps_climo - crps_fcst)/crps_climo;
-      }
-      else {
-         it->second.ens_pd.crpss = bad_data_double;
-      }
+      // Compute aggregated empirical CRPSS
+      it->second.ens_pd.crpss_emp =
+         (is_bad_data(crps_emp) || is_bad_data(crpscl_emp) && is_eq(crpscl_emp, 0.0) ?
+         bad_data_double : (crpscl_emp - crps_emp)/crpscl_emp);
+
+      // Compute aggregated Gaussian CRPSS
```

Comment:
I think this is supposed to be `it->second.ens_pd.crpss_gaus` ?

Reply:
Indeed it is. Thanks for catching that bug and letting me know. I just updated it.